crates/flox/src/commands/environment.rs

```diff
@@ -32,640 +31,719 @@ pub struct EnvironmentArgs {
-                            error!("Environment invalid, building resulted in an error: {e}");
-                            let again = Dialog {
-                                message: "Continue editing?",
-                                help_message: Default::default(),
-                                typed: Confirm {
-                                    default: Some(true),
-                                },
-                            }
-                            .prompt()
-                            .await?;
-                            if !again {
-                                break 'edit;
-                            }
-                        },
-                    };
-                }
-                environment
-                    .replace_with(temporary_environment)
-                    .context("Failed applying environment changes")?;
-            },
-
-            EnvironmentCommands::Destroy { environment, .. } if !Feature::Env.is_forwarded()? => {
-                let environment =
-                    resolve_environment(&flox, environment.as_deref(), "install").await?;
-
-                environment
-                    .delete()
-                    .context("Failed to delete environment")?;
-            },
+// impl EnvironmentCommands {
+//     pub async fn handle(&self, flox: Flox) -> Result<()> {
+//         match self {
+//             EnvironmentCommands::List { .. } => subcommand_metric!("list"),
+//             EnvironmentCommands::Envs => subcommand_metric!("envs"),
+//             EnvironmentCommands::Activate { .. } => subcommand_metric!("activate"),
+//             EnvironmentCommands::Init { .. } => subcommand_metric!("init"),
+//             EnvironmentCommands::Destroy { .. } => subcommand_metric!("destroy"),
+//             EnvironmentCommands::Edit { .. } => subcommand_metric!("edit"),
+//             EnvironmentCommands::Export { .. } => subcommand_metric!("export"),
+//             EnvironmentCommands::Generations { .. } => subcommand_metric!("generations"),
+//             EnvironmentCommands::Git { .. } => subcommand_metric!("git"),
+//             EnvironmentCommands::History { .. } => subcommand_metric!("history"),
+//             EnvironmentCommands::Import { .. } => subcommand_metric!("import"),
+//             EnvironmentCommands::Install { .. } => subcommand_metric!("install"),
+//             EnvironmentCommands::Push { .. } => subcommand_metric!("push"),
+//             EnvironmentCommands::Pull { .. } => subcommand_metric!("pull"),
+//             EnvironmentCommands::Uninstall { .. } => subcommand_metric!("remove"),
+//             EnvironmentCommands::Rollback { .. } => subcommand_metric!("rollback"),
+//             EnvironmentCommands::SwitchGeneration { .. } => subcommand_metric!("switch"),
+//             EnvironmentCommands::Upgrade { .. } => subcommand_metric!("upgrade"),
+//             EnvironmentCommands::WipeHistory { .. } => subcommand_metric!("wipe-history"),
```

Comment:
Are these added back in somewhere?

Reply:
good spot, no not yet

Reply:
done in [d6f7167](https://github.com/flox/flox/pull/215/commits/d6f7167891272682b26dab5b315f2fe7babe839b)
and
[ed94d3a](https://github.com/flox/flox/pull/215/commits/ed94d3ae9075a0abb0e6836f0ac115689ae495ef)
for forwarded commands

Reply:
Can we drop the commented out stuff now?